src/js/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,258 @@
+	margin-bottom: 16px;
+`;
+
+const ProductDescription = styled.div`
+	font-size: 18px;
+	line-height: 26px;
+	color: #828282;
+	margin-bottom: 16px;
+`;
+
+const ProductPrice = styled.div`
+	display: flex;
+	flex-direction: row;
+	align-items: flex-end;
+	padding: 0px;
+	height: 35px;
+	margin-bottom: 24px;
+`;
+
+const ProductPrice1 = styled.div`
+	font-weight: bold;
+	font-size: 24px;
+	line-height: 35px;
+	margin-right: 8px;
+`;
+const ProductPrice2 = styled.div`
+	font-size: 16px;
+	line-height: 23px;
+	text-decoration-line: line-through;
+	color: #828282;
+`;
+const Line = styled.div`
+	width: 440px;
+	height: 1px;
+	background: #e0e0e0;
+	margin: 8px 0px;
+`;
+const Number = styled.div`
+	position: relative;
+	width: 440px;
+	height: 41px;
+	display: flex;
+	align-items: center;
+	margin: 16px 0px;
+`;
+const NumberLabel = styled.div`
+	width: 60px;
+	height: 23px;
+	font-size: 16px;
+	line-height: 23px;
+	color: #828282;
+`;
+const NumberInput = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	position: absolute;
+	width: 57px;
+	height: 41px;
+	top: 0px;
+	left: 354px;
+	border: 1px solid #e0e0e0;
+`;
+const NumberUpButton = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	position: absolute;
+	width: 28px;
+	height: 20px;
+	left: 412px;
+	top: 0px;
+	border: 1px solid #e0e0e0;
+`;
+const NumberDownButton = styled.div`
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	position: absolute;
+	width: 28px;
+	height: 20px;
+	left: 412px;
+	top: 21px;
+	border: 1px solid #e0e0e0;
+`;
+const TotalPrice = styled.div`
+	display: flex;
+	flex-direction: row;
+	justify-content: flex-end;
+	align-items: center;
+	padding: 0px;
+
+	position: absolute;
+	width: 360px;
+	height: 46px;
+	left: 552px;
+	top: 496px;
+`;
+const TotalPriceLabel = styled.div`
+	height: 26px;
+	font-weight: bold;
+	font-size: 18px;
+	line-height: 26px;
+	text-align: right;
+	color: #828282;
+`;
+const TotalPriceContent = styled.div`
+	height: 46px;
+	font-weight: bold;
+	font-size: 32px;
+	line-height: 46px;
+	text-align: right;
+	color: #010101;
+	margin-left: 24px;
+`;
+const CloseButton = styled.div`
+	position: absolute;
+	width: 32px;
+	height: 32px;
+	left: 968px;
+	top: 0px;
+	svg {
+		position: absolute;
+	}
+`;
+const ButtonWrapper = styled.div`
+	position: absolute;
+	top:574px;
+	left:472px;
+`;
+
+const Modal = ({ data, isModalOn, setModalOn }) => {
+	const [count, setCount] = useState(1);
+	const price = parseInt((data.prices[1] ? data.prices[1] : data.prices[0]).replace("원", "").replace(",", ""));
+	return (
+		<ModalWrapper isOn={isModalOn}>
+			<Box>
+				<ThumbNail {...data} />
+				<ProductInfo>
+					<ProductName>{data.name}</ProductName>
+					<ProductDescription>{data.product_description}</ProductDescription>
+					<ProductPrice>
+						{data.badge ? <Badge badge={data.badge} /> : ""}
+						<ProductPrice1>{data.prices[1] ? data.prices[1] : data.prices[0]}</ProductPrice1>
```

Comment:
p5) 개인취향이긴한데 삼항연산자 말고 or로 처리할 수 도 있겠네요 ㅎㅎ
```suggestion
						<ProductPrice1>{data.prices[1] || data.prices[0]}</ProductPrice1>
```